notebooks/04_model_select_and_optimize/tuning_spark_als.ipynb

```diff
@@ -37,16 +37,16 @@
   },
```

Comment:
<p>I think we should use %matplotlib notebook instead of inline, it seems to work nicer for me</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/864/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>